daru-io.gemspec

```diff
@@ -20,6 +27,37 @@ Gem::Specification.new do |spec|
   spec.executables   = spec.files.grep(%r{^bin/}) { |f| File.basename(f) }
   spec.require_paths = ['lib']
 
+  spec.post_install_message = <<-EOF
+
+*************************************************
+|        Thank you for installing daru-io!      |
+|                                               |
+|                      //                       |
+|                     //                        |
+|                    ||                         |
+|                    ||                         |
+|                    ||                         |
+|                oOOOOOOOOOo                    |
+|               ,         Oo                    |
+|              //|         |                    |
+|              \\\\|         |                    |
+|                | --- .-. |                    |
+|                |  |  | | |                    |
+|                | _|_ ._. |                    |
+|                `---------`                    |
+|                                               |
+|  Hope you love using daru-io! Also, consider  |
+|  using daru-view for various visualizations   |
+|  such as Tables and Charts plotting. Read     |
+|  the README for interesting use cases and     |
+|  examples.                                    |
+|                                               |
+|  Cheers!                                      |
+|  SciRuby Team                                 |
+*************************************************
+
+EOF
```

Comment:
And this is how daru-io looks on installation. Added a straw to the booze to literally show "Export" from "daru". 😉 

![image](https://user-images.githubusercontent.com/17109060/29145817-3491c8de-7d7c-11e7-984f-4ecb3914af0d.png)